From 33dc40a7e194f62d091c1d91147376f91f3555e1 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 27 Dec 2017 20:25:58 +0800
Subject: [PATCH] MLK-17314-2 arm: imx: fix build warning

Fix build warning introduced by below commit:
(556d2d5 MLK-16750-5: arm: imx: support using psci to handle power stuff)

arch/arm/mach-imx/pm-imx7ulp.c: In function 'imx7ulp_pm_common_init':
arch/arm/mach-imx/pm-imx7ulp.c:747:17: warning:
'sram_paddr' may be used uninitialized in this function [-Wuninitialized]

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
---
 arch/arm/mach-imx/pm-imx7ulp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/pm-imx7ulp.c b/arch/arm/mach-imx/pm-imx7ulp.c
index 86c17cc..e6a77c6 100644
--- a/arch/arm/mach-imx/pm-imx7ulp.c
+++ b/arch/arm/mach-imx/pm-imx7ulp.c
@@ -587,7 +587,7 @@ void __init imx7ulp_pm_common_init(const struct imx7ulp_pm_socdata
 				*socdata)
 {
 	struct device_node *np;
-	unsigned long sram_paddr;
+	unsigned long sram_paddr = 0;
 	const u32 *mmdc_offset_array;
 	const u32 *mmdc_io_offset_array;
 	unsigned long i, j;
-- 
2.9.2

